cmd/vm_stop.go

```diff
@@ -8,10 +8,9 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// stopCmd represents the stop command
 var vmStopCmd = &cobra.Command{
-	Use:               "stop <vm name | id>+",
-	Short:             "Stop virtual machine instance",
+	Use:               "stop NAME|ID",
+	Short:             "Stop a running Compute instance",
```

Comment:
Same remark as above:
```suggestion
	Short:             "Stop a Compute instance",
```